packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -110,15 +125,45 @@ class GameWidget<T extends Game> extends StatefulWidget {
     this.focusNode,
     this.autofocus = true,
     MouseCursor? mouseCursor,
-  }) : super(key: key) {
+  })  : create = null,
+        super(key: key) {
     if (mouseCursor != null) {
-      game.mouseCursor = mouseCursor;
+      game!.mouseCursor = mouseCursor;
     }
     if (initialActiveOverlays != null) {
-      game.overlays.addAll(initialActiveOverlays);
+      game!.overlays.addAll(initialActiveOverlays);
     }
   }
 
+  /// Renders a [game] in a flutter widget tree.
+  ///
+  /// Unlike the default constructor, this creates a [GameWidget] that controls
+  /// the creation of the game instance, removing the necessity of rendering
+  /// this widget inside a [StatefulWidget].
```

Comment:
I wouldn't call this a "necessity" though, since the simplest way to use a `GameWidget` is to write
```dart
void main() {
  final game = MyGame();
  return GameWidget(game: game);
}
```
So perhaps this could be stated as a use-case geared towards more complicated widget trees?

Reply:
Updated the dartdocs and added the `runApp` method (which I guess you meant) to the game widget docs.